malaria-ios/ViewControllers/Constants.swift

```diff
@@ -1,9 +1,26 @@
-class Constants {
+import UIKit
 
+class Constants {
+  
   struct Widget {
     static let BundleID = "anita-borg.pc-malaria-ios.pill-widget"
     static let DidTakePillForToday = "DidTakePillForToday"
     static let AppGroupBundleID = "group.anita-borg.pc-malaria-ios.took-pill"
     static let FirstRunFlag = "FirstRunFlag"
+    
+    // User Profile
+    static let RemainingPills = "Remaining Pills Constant"
+    static let RemainingPillsUnitMeasure = "Remaining Pills Unit Measure Constant"
+  }
+  
+  struct UserProfile {
+    static let CellReuseIdentifier = "User Profile Pill Cell Identifier"
+    static let PillReminderValue = "Pill Reminder Value"
```

Comment:
Are you using this variable? I think not as you are using in the UserSettings enum (which is great).

If you are not using, remove.